app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md

```diff
@@ -0,0 +1,321 @@
+---
+title: "Training fees and financial support for non-UK trainee teachers"
+description: |-
+  Learn more about bursaries, scholarships and the international relocation payment.
+related_content:
+    Career progression stories: "/blog/tag/career-progression"
+    Salaries and benefits of teaching in England : "/salaries-and-benefits"
+    A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
+promo_content:
+date: "2021-05-27"
+image: "media/images/content/hero-images/0002.jpg"
+backlink: "../../"
+keywords:
+  - International
+  - Overseas
+  - Foreign
+  - Nationality
+  - Residency
+  - EEA
+  - Canada
+  - New Zealand
+  - Switzerland
+  - USA
+  - Visa
+  - Skilled worker visa
+  - EU
+  - EU Settlement Scheme
+  - NARIC
+  - UK NARIC
+  - ENIC
+  - UK ENIC
+  - European Network of Information Centre
+  - National Recognition Information Centre for the United Kingdom
+  - PhD
+  - Immigration
+  - Ireland
+  - Irish
+  - Common Travel Area
+  - Student Visa
+  - Home Office
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Japan
+  - Monaco
+  - Hong Kong
+  - Republic of Korea
+  - Taiwan
+  - Family Visa
+  - UK Ancestry Visa
+  - Abroad
+  - Europe
+  - European
+  - Settlement Scheme
+---
+
+If you’re applying for teacher training as a non-UK citizen, you may be able to get some financial help from the UK government. The information on this page is for:
+
+
+* non-UK citizens living in the UK
+* non-UK citizens living outside the UK
+* Irish citizens
+* refugees or asylum seekers
+
+## Teacher training fees
+
+As a non-UK candidate, you can expect to pay between £9,250 and about £32,000 to train to teach in England – fees vary between training providers. You’ll also have to cover your living costs.
```

Comment:
```suggestion
As a non-UK candidate, you can expect to pay between £9,250 and about £32,000 to train to teach in England – fees vary between training providers.
```
Because this section is talking about teacher training fees, I wonder if we need that last sentence?